src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -114,10 +107,14 @@ class OPENVINO_GENAI_EXPORTS LLMPipeline {
     */
     LLMPipeline(
         const std::string& path,
-        const std::string& device="CPU",
-        const ov::AnyMap& plugin_config={}
+        const std::string& device,
+        const ov::AnyMap& plugin_config = {}
     );
 
+    OPENVINO_DEPRECATED("Please, specify device explicitly when create LLMPipeline")
```

Comment:
How do we track when to remove it? Right after 24.5 because it's LTS?

Reply:
added that it will be removed after 25.0